pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -287,6 +287,11 @@ func addSingleNumberDataPoint(pt pdata.NumberDataPoint, resource pdata.Resource,
 // addSingleHistogramDataPoint converts pt to 2 + min(len(ExplicitBounds), len(BucketCount)) + 1 samples. It
 // ignore extra buckets if len(ExplicitBounds) > len(BucketCounts)
 func addSingleHistogramDataPoint(pt pdata.HistogramDataPoint, resource pdata.Resource, metric pdata.Metric, settings Settings, tsMap map[string]*prompb.TimeSeries) {
+	if !pt.HasSum() {
+		// If the sum is unset, it indicates the _sum metric point should be
+		// omitted
+		return
+	}
```

Comment:
This function does more than just creating the `_sum` point.  I think this would result in omitting any histogram that does not have a sum.  Is that the intent?  The comment should be updated to reflect that if it is.

Reply:
Fixed.  It should have been caught by the unit test, but I forgot that the unit tests still aren't being run: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/8401.

I tested it with those changes, and it does the correct thing now.